src/workflows/test_workflows/annotation/scgpt_knn_test/script.py

```diff
@@ -0,0 +1,40 @@
+from mudata import read_h5mu
+import shutil
+import os
+import sys
+from pathlib import Path
+import pytest
+
+##VIASH START
+par = {
+    "input": "input.h5mu"
+}
+
+meta = {
+    "resources_dir": "resources_test"
+}
+##VIASH END
+
+
+def test_run():
+    input_mudata = read_h5mu(par["input"])
```

Comment:
Could you put the input_mudata in a separate pytest fixture? This way if any other extra tests are added, the mudata object is cached.